Examples/CaseStudies/SwiftUICaseStudies/01-GettingStarted-Bindings-Basics.swift

```diff
@@ -35,24 +35,26 @@ struct BindingBasics: Reducer {
     case toggleChanged(isOn: Bool)
   }
 
-  func reduce(into state: inout State, action: Action) -> Effect<Action> {
-    switch action {
-    case let .sliderValueChanged(value):
-      state.sliderValue = value
-      return .none
-
-    case let .stepCountChanged(count):
-      state.sliderValue = .minimum(state.sliderValue, Double(count))
-      state.stepCount = count
-      return .none
-
-    case let .textChanged(text):
-      state.text = text
-      return .none
-
-    case let .toggleChanged(isOn):
-      state.toggleIsOn = isOn
-      return .none
+  var body: some Reducer<State, Action> {
```

Comment:
Are you writing `Reducer<State, Action>` because it's an example? Self can be used here as well right? If `ReducerOf` is used

```suggestion
  var body: some ReducerOf<Self> {
```

Reply:
@Muhammed9991 We haven't 100% landed on a style here. While we like the brevity of `ReducerOf<Self>` and it can be used most of the time, it is another concept to be aware of, so we need to decide how and when to disclose it...